src/java.desktop/share/classes/sun/java2d/marlin/ArrayCacheDouble.java

```diff
@@ -38,27 +45,23 @@
 import sun.java2d.marlin.ArrayCacheConst.CacheStats;
 
 /*
- * Note that the [BYTE/INT/FLOAT/DOUBLE]ArrayCache files are nearly identical except
+ * Note that the ArrayCache[BYTE/INT/FLOAT/DOUBLE] files are nearly identical except
  * for a few type and name differences. Typically, the [BYTE]ArrayCache.java file
  * is edited manually and then [INT/FLOAT/DOUBLE]ArrayCache.java
  * files are generated with the following command lines:
```

Comment:
Looks like here and in other places this comment became outdated since the "command lines" below are removed?

Reply:
You're right, I will fix that comment in all ArrayCacheXXX files.